### DISCUSSION TITLE
Analytics / Vitals blocked by adblocker

### DISCUSSION THREAD

**Discussion by jonathanstanley** [vercel/community#1055] [Help] [answered]
Turning on analytics (with Svelte app) and found the following in the network monitor:

```
Blocked Requests: vitals
Status: (blocked:other)
General:
Request URL: https://vitals.vercel-analytics.com/v1/vitals
Referrer Policy: strict-origin-when-cross-origin
```

The problem is the ad blocker. I turned off the adblocker and it worked ([this post prompted me to check](https://github.com/vercel/next.js/issues/18361#issuecomment-939522497))

It is a bit confusing because Vercel says requests are sent to the same domain:
<img width="384" alt="Screenshot 2022-11-15 at 1 03 53 PM" src="https://user-images.githubusercontent.com/2755958/201993341-58f28bb3-b6b2-4bcc-8c2d-5722327cc210.png">

But the Svelte example doesn't send it to the same domain:
https://github.com/vercel/vercel/blob/4e41c0e0a60fa443e0a0b2a9aba539d02c9ffc10/examples/sveltekit/src/lib/vitals.js#L3

Is the marketing page wrong? did I miss something?

**Comment by amyegan** [comment, marked as answer]
Good question @jonathanstanley! This example is a little older and does use a different domain to collect Web Vitals, as you saw. 

Vercel Analytics has two parts. One is the older [Web Vitals](https://vercel.com/docs/concepts/analytics/web-vitals#) feature, and the other is a newer [Audiences](https://vercel.com/docs/concepts/analytics/audiences#) feature which is still in Beta. The screenshot you shared is part of the  Audiences real-time traffic section of the [Vercel Analytics](https://vercel.com/analytics) features page. 

Web Vitals uses the [Web Vitals API](https://vercel.com/docs/concepts/analytics/api) which may be caught by adblockers. The Audiences `@vercel/analytics` package works a little differently so the domain used matches your site and adblockers don't notice. You can follow the [Audiences Quickstart guide](https://vercel.com/docs/concepts/analytics/audiences/quickstart) to it up for your site.

I agree that the wording on the marketing page is a little confusing and I'm sharing your feedback with the team! 🙇
